Sources/MapboxMaps/Location/Puck/Puck3D.swift

```diff
@@ -89,15 +95,69 @@ internal final class Puck3D: Puck {
             try! style.addSource(source, id: Self.sourceID)
         }
 
+        // Mercator scale
+        maybeUpdateMercatorScale(at: location.coordinate.latitude)
+
         // create the layer if needed
         if !style.layerExists(withId: Self.layerID) {
```

Comment:
There seems to be a problem here — the first time this code runs, there could be an attempt to update the layer before it is created.